lib/archiving.rb

```diff
@@ -27,11 +27,16 @@ def self.archive(date)
           end
         end
       end
+
       puts "Removing archived data from database for #{date}..."
       deliveries.find_each do |delivery|
         delivery.destroy
       end
+
       copy_to_s3(date)
+
+      puts "Removing temp local file #{date}.tar.gz copied to S3..."
+      File.delete("db/archive/#{date}.tar.gz")
```

Comment:
Am I right in thinking that if you don't have S3 configured, or there's a problem with it, that this archive method will actually just irretrievably delete stuff from the database? That doesn't sound like a good thing and definitely not what you'd expect a method called "archive" to do!